library/src/test/java/com/mindera/skeletoid/logs/appenders/LogFileAppenderUnitTest.java

```diff
@@ -0,0 +1,131 @@
+package com.mindera.skeletoid.logs.appenders;
+
+import android.content.Context;
+
+import com.mindera.skeletoid.logs.LOG;
+
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class LogFileAppenderUnitTest {
+
+    private String PACKAGE_NAME = "my.package.name";
+    private final String FILE_NAME = "FILE_NAME";
+
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConstructorAllNull() {
+        LogFileAppender appender = new LogFileAppender(null, null);
+
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConstructorPackageNameNull() {
+        LogFileAppender appender = new LogFileAppender(null, FILE_NAME);
+
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConstructorFileNameInvalid() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, "*");
+
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConstructorFileNameNull() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, null);
+
+    }
+
+    @Test
+    public void testFileNameIsValidEmpty() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, FILE_NAME);
+        assertFalse(appender.isFilenameValid(""));
+    }
+
+    @Test
+    public void testFileNameIsValidInvalid() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, FILE_NAME);
+        assertFalse(appender.isFilenameValid("*"));
+    }
+
+    @Test
+    public void testFileNameIsValid() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, FILE_NAME);
+        assertTrue(appender.isFilenameValid(FILE_NAME));
+    }
+
+
+    @Test
+    public void testConstructor() {
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, FILE_NAME);
+
+        assertEquals("LogFileAppender", appender.getLoggerId());
+    }
+
+    @Test
+    public void testEnableAppender() {
+        Context context = mock(Context.class);
+
+        LogFileAppender appender = new LogFileAppender(PACKAGE_NAME, FILE_NAME);
+
+        appender.enableAppender(context);
+
+        //TODO What should I validate here?
+        verify(appender, times(1)).enableAppender(context);
```

Comment:
I'm also not sure what should be checked here.
But this kind of check doesn't make sense, I think 🤔
What about checking (somehow) the values in `mFileHandler`, `mCanWriteToFile` and `mFileLoggingTP`?